en/docs/using-wso2-identity-server/username-recovery.md

```diff
@@ -33,22 +33,28 @@ with the WSO2 IS.
     compatitbility and can still be used if required.
     
     
-    Before you begin
+!!! tip "Before you begin"
     
     Ensure that the " `         IdentityMgtEventListener        ` " with the
     `         orderId=50        ` is set to **false** and that the Identity
     Listeners with `         orderId=95        ` and
     `         orderId=97        ` are set to **true** in the
-    `         <IS_HOME>/repository/conf/identity/identity.xml        ` file.
+    `         <IS_HOME>/repository/conf/deployment.toml       ` file.
     
     ``` java
-    <EventListener type="org.wso2.carbon.user.core.listener.UserOperationEventListener" name="org.wso2.carbon.identity.mgt.IdentityMgtEventListener" orderId="50" enable="false"/>
-    <EventListener type="org.wso2.carbon.user.core.listener.UserOperationEventListener" name="org.wso2.carbon.identity.governance.listener.IdentityMgtEventListener" orderId="95" enable="true" />
-    <EventListener type="org.wso2.carbon.user.core.listener.UserOperationEventListener" name="org.wso2.carbon.identity.governance.listener.IdentityStoreEventListener" orderId="97" enable="true">
-```
+    [event.default_listener.identity_mgt]
+    priority= "50"
+    enable = false
+    [event.default_listener.governance_identity_mgt]
+    priority= "95"
+    enable = true
+    [event.default_listener.governance_identity_store]
+    priority= "97"
+    enable = true
+    ```
 
 
-1.  Configure the following email settings in the \<
+1.  Configure the following email settings in the <
     `           IS_HOME>/repository/conf/output-event-adapters.xml          `
```

Comment:
toml config?

Reply:
This has not been templated yet. Will create an issue with a few other configs.